pkg/shardddl/optimism/lock.go

```diff
@@ -119,16 +128,11 @@ func (l *Lock) TrySync(callerSource, callerSchema, callerTable string,
 			}
 		}
 	}
-	l.joined = newJoined // update the current table info.
-	log.L().Info("update joined table info", zap.String("lock", l.ID), zap.Stringer("from", oldJoined), zap.Stringer("to", newJoined),
```

Comment:
Why do we delete this log?

Reply:
table info often too long, and if the joined table info changed it will be logged later, so I deleted this duplicate one.